notify/__init__.py

```diff
@@ -11,4 +11,4 @@
 )
 
 name = "notify"
-__version__ = "0.5.1"
+__version__ = "0.6.0"
```

Comment:
Because we need a new version of the webhook, i think it is best to make this a major release. The old teams_webhook will not work with this setup